pkg/controllers/edgeconnect/deployment/deployment_test.go

```diff
@@ -66,6 +66,88 @@ func Test_buildAppLabels(t *testing.T) {
 	})
 }
 
+func TestLabels(t *testing.T) {
+	testLabelKey := "test-label-key"
+	testLabelValue := "test-label-value"
+
+	t.Run("Check empty custom labels", func(t *testing.T) {
+		ec := &edgeconnect.EdgeConnect{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      testName,
+				Namespace: testNamespace,
+			},
+			Spec: edgeconnect.EdgeConnectSpec{},
+		}
+
+		deployment := New(ec)
+
+		assert.Len(t, deployment.Spec.Template.Labels, 5)
```

Comment:
This check a bit confusing, better IMO demonstrate that we only have default app labels.